src/symbol_table.rs

```diff
@@ -32,9 +32,14 @@ impl SymbolTable {
 
     // Interns the v1.0 system symbols
     pub(crate) fn initialize(&mut self) {
-        for &text in v1_0::SYSTEM_SYMBOLS.iter() {
-            self.intern_or_add_placeholder(text);
-        }
+        self.add_placeholder(); // $0
+        v1_0::SYSTEM_SYMBOLS[1..]
+            .iter()
+            .copied()
+            .map(Option::unwrap)
+            .for_each(|text| {
+                let _sid = self.add_symbol_for_text(text);
+            });
```

Comment:
🗺️ The changes in this file are to offer method versions that accept a `Symbol` instead of a `&str`. This allows the reader to heap-allocate once per symbol definition instead of twice.